src/modules/main/store/actions.ts

```diff
@@ -20,7 +21,9 @@ export function loadNewest(payload: INewestItem[]): MainActions {
   };
 }
 
-export function loadExperts(payload: IExpertsItem[]): MainActions {
+export function loadExperts(): MainActions {
+  const payload = cards;
```

Comment:
please use capital case `CARDS` - for global constants (mocks fit this category)